can_tools/scrapers/official/TX/texas_vaccine.py

```diff
@@ -142,11 +145,14 @@ class TXVaccineCountyAge(TexasVaccineParent):
     sheet_name = "By County, Age"
     replacers = {
         "age": {
-            "12-15 years": "12-15",
-            "16-49 years": "16-49",
-            "50-64 years": "50-64",
-            "65-79 years": "65-79",
-            "80+ years": "80_plus",
+            # Excel automatically parses some of the string entries as dates, and
+            # read excel does not seem to have a way to fix this.
+            # This is not a good fix, but it's the only way I could find to convert the
+            # date values back into strings.S
+            # See https://github.com/pandas-dev/pandas/issues/39898 for explanation
+            "2021-05-11 00:00:00": "5-11",
```

Comment:
Excel was doing some fun excel stuff here with assuming the datatypes--it interpreted `12-15` and `5-11` as dates. I did a bit of digging but couldn't find a real fix, so I opted for this kinda' gross solution. 

This issue is related, but didn't provide a good solution: https://github.com/pandas-dev/pandas/issues/39898

Reply:
Heh, yikes.  Thanks for the comment.